package-res/resources/web/pentaho/data/_filter/AbstractFilter.js

```diff
@@ -79,9 +79,20 @@ define([
       return _apply(this, dataTable);
     },
 
-    transform: function(iteratee){
+    /**
```

Comment:
Not sure `walk` is the usual term for a traversal function that rewrites or transforms the tree. I saw this on MSDN: https://msdn.microsoft.com/en-us/library/bb882521(v=vs.90).aspx. Please take a look, even seeing if the iteratee contract is similar to ours, and if any better method exists :-)